awscli/utils.py

```diff
@@ -277,11 +291,21 @@ def get_pager_stream(self, preferred_pager=None):
 
     @contextlib.contextmanager
     def get_stdout_stream(self):
-        yield get_binary_stdout()
+        yield get_stdout_text_writer()
```

Comment:
What's the context for this change?

Reply:
Most of the output through the CLIDriver is written out as text. The goal of this was to unify the output stream factory to always produce text streams. I think before when I wrote this class for the history commands, I did not realize that I could force the stdin for Popen to be text with `universal_newlines` and so I just went with using binary streams. If I did not make this change, I would have to either not reuse this class or make all of the CLI formatters to write out binary, which would be a larger change. Does that make more sense?

Reply:
Cool, makes sense.